src/tree/workspace/WorkspaceResourceTreeDataProvider.ts

```diff
@@ -47,7 +47,7 @@ export class WorkspaceResourceTreeDataProvider extends ResourceTreeDataProviderB
 
             // NOTE: Returning zero children indicates to VS Code that is should display a "welcome view".
             //       The one chosen for display depends on the context set above.
-            return children;
+            return children.sort((a, b) => a.id.localeCompare(b.id));
```

Comment:
I was wondering-- should we separate the workspace items by which extension contributed them? Alphabetical order may not make that much sense,

Reply:
I was thinking that sorting by id would be most flexible. We can manage the ids as needed to sort it how we want.

Reply:
That's fair enough.